src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -509,7 +509,7 @@ export default class ArrayField extends React.Component {
             disabled={!this.props.formData || addAnotherDisabled}
             onClick={this.handleAdd}
           >
-            Add another {uiOptions.itemName}
+            Add another {(uiOptions.itemName || 'item').toLowerCase()}
```

Comment:
this references `uiOptions.itemName` rather than just `itemName` because it's outside of the mapper where we have access to `itemName`